src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const NORMAL_PRICE = 40;
+  const MID_TERM = 3;
+  const MID_TERM_DISC = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISC = 50;
+
+  let finalPrice = days * NORMAL_PRICE;
+
+  if (days >= LONG_TERM) {
+    return (finalPrice -= LONG_TERM_DISC);
```

Comment:
The parentheses around the assignment `finalPrice -= LONG_TERM_DISC` are unnecessary and can be omitted. Simply return `finalPrice - LONG_TERM_DISC`.